Kernel/API/POSIX/unistd.h

```diff
@@ -38,6 +38,7 @@ enum {
     _SC_PAGESIZE,
     _SC_GETPW_R_SIZE_MAX,
     _SC_CLK_TCK,
+    _SC_SYMLOOP_MAX
```

Comment:
This should probably have a trailing `,`, even if it's not required by the syntax.

Reply:
Added